bundles/org.openhab.binding.nibeuplink/src/main/resources/OH-INF/i18n/nibeuplink.properties

```
@@ -467,3 +467,9 @@ channel-type.nibeuplink.type-switch.label = Unnamed Switch
 channel-type.nibeuplink.type-temperature.label = Unnamed Temperature
 channel-type.nibeuplink.type-time-scale10.label = Unnamed Time
 channel-type.nibeuplink.type-time-unscaled.label = Unnamed Time
+
+# status translations
+
+status.invalid.nibeId = "most likely your NibeId is wrong. please check your NibeId"
+status.invalid.credentials = "invalid username or password"
+status.online = "logged in"
```

```suggestion
status.online = "Logged in"
```
(if preserved, but see other comment)

---

will remove it